collectors/framework/models.py

```diff
@@ -181,7 +181,7 @@ class CollectorState(models.TextChoices):
     @property
     def crontab_params(self) -> Optional[Dict[str, str]]:
         if self.crontab:
-            params = re.search(r"<crontab: (.*) \(m/h/d/dM/MY\)>", self.crontab).group(
+            params = re.search(r"<crontab: (.*) \(m/h/dM/MY/d\)>", self.crontab).group(
```

Comment:
What about existing CollectorMetadata objects? I think most of the crontabs we use only use minute/hour but if not, the string representation in the database should be migrated

Reply:
Good observation, I need to add the migration

Reply:
I theoretically does not matter as we only use hours and minutes which do not change but rather to be safe